src/Symfony/Component/ErrorHandler/Debug.php

```diff
@@ -58,3 +58,5 @@ public static function enable($errorReportingLevel = E_ALL, $displayErrors = tru
         DebugClassLoader::enable();
     }
 }
+
+class_alias('Symfony\Component\ErrorHandler\Debug', 'Symfony\Component\Debug\Debug');
```

Comment:
`if (!class_exists('Symfony\Component\Debug\Debug', false))`? Trying to alias to a known class will fail.

Reply:
Kind of related : why don't we mark the new Error components as conflicting with the Debug one so there can't be both at the same time?

Reply:
> Kind of related : why don't we mark the new Error components as conflicting with the Debug one so there can't be both at the same time?

We need both components installed to be able to migrate, don't we?

Reply:
There is no need to migrate if you already require the new Error component I guess

Reply:
We cannot conflict with the Debug component - that'd be artificial + wouldn't make sense on 4.4 where symfony/symfony replaces both.